Source/WebKit/Platform/IPC/StreamClientConnection.h

```diff
@@ -139,8 +139,10 @@ class StreamClientConnection final : public ThreadSafeRefCounted<StreamClientCon
         void didClose(Connection&) final;
         void didReceiveInvalidMessage(Connection&, MessageName, int32_t indexOfObjectFailingDecoding) final;
     private:
+        Ref<Connection::Client> protectedReceiver() { return m_receiver.get(); }
+
         CheckedRef<StreamClientConnection> m_owner;
-        Connection::Client& m_receiver;
+        WeakRef<Connection::Client> m_receiver;
```

Comment:
We could use `const CheckedRef<>` here I believe and then we wouldn't need `protectedReceiver()` no?

Reply:
Good point! Initially, this patch was changing the raw reference to `WeakPtr`, so I recommended `WeakRef`. Then, since that isn't ref'ing, I thought making it `const` won't do allow us to eliminate `protectedReceiver`. But `const CheckedRef` should.

@annevk This is the right change to make here.